index.html

```diff
@@ -49,6 +49,7 @@
     <div id='nav'>
       <div class='container center'>
         <a href="#contribute">Contribute</a>
+        <a href="#ci">CI</a>
```

Comment:
I'm not opposed to adding CI to the navbar, but I think it's the straw that broke the camels back (and only because the camel should've been fixed a long time ago). This is what the navbar looks like in fullscreen on my 13" display:
<img width="1280" alt="screen shot 2017-04-09 at 21 27 12" src="https://cloud.githubusercontent.com/assets/321790/24840325/685a5008-1d6b-11e7-8924-734e5e431a08.png">
To fix I agree with @schubert, so I pushed a PR to do just that: #28. That one also removes the Tutorial link from the navbar since it's already prominently display right above the navbar. While the site could do with a makeover to transition into the world where 5.0 has been released as open-source, I think it's a decent stepping stone. So, +1 on adding CI combined with #28 (or a similar patch).